### DISCUSSION TITLE
网页端的配置修改功能打开是一片空白页啊，其他的功能有内容

### DISCUSSION THREAD

**Discussion by httpsColon** [Pik-4/HsMod#132] [Q&A]
http://[127.0.0.1:58744/config/index.html](http://127.0.0.1:58744/config/index.html)

**Comment by lomk-pig** [comment]


**Reply by httpsColon** [reply]
天哪，我一直以为是放到BepInEx\config

**Comment by Pik-4** [comment]
没有配置管理，哪来的配置界面

**Reply by httpsColon** [reply]
放错位置了，以为是BepInEx\config
